source/_integrations/intellifire.markdown

```diff
@@ -0,0 +1,34 @@
+---
+title: Intellifire 
+description: Instructions on the Intellifire Fireplace integration for Home Assistant.
+ha_category:
+  - Binary Sensor
+ha_iot_class: Local Polling
+ha_release: 0.47
+ha_codeowners:
+  - '@jeeftor'
+ha_domain: intellifire
+ha_config_flow: true
+---
+
+
+### Overview
+
+Intellifire Wifi fireplace modules provide app-based and Alexa control to various fireplace. The modules do expose an unencrypted HTTP endpoint on the network that provides status information. This integration will read that URL and create a set of sensors displaying the current fireplace state.
```

Comment:
```suggestion
IntelliFire Wi-Fi fireplace modules provide app-based and Alexa control to various fireplaces. The modules do expose an unencrypted HTTP endpoint on the network that provides status information. This integration will read that URL and create a set of sensors displaying the current fireplace state.
```

I'm not sure if we need to explain the unencrypted connection and the URL read and such. Is it really useful for an end-user user to tell about how it works internally?